File: src/main/java/cz/kamenitxan/jakon/core/customPages/CustomPage.java
package cz.kamenitxan.jakon.core.customPages;

import cz.kamenitxan.jakon.core.model.JakonObject;

/**
 * Created by TPa on 26.04.16.
 */
public abstract class CustomPage extends JakonObject {
	public void render() {
		throw new UnsupportedOperationException("Render method for class " + this.getClass().getSimpleName()
												+ " is not implemented");
	}
}

File: src/main/java/cz/kamenitxan/jakon/core/customPages/CustomPage.scala
package cz.kamenitxan.jakon.core.customPages

import cz.kamenitxan.jakon.core.controler.IControler
import cz.kamenitxan.jakon.core.model.JakonObject

/**
  * Created by TPa on 26.04.16.
  */
abstract class CustomPage extends JakonObject(childClass = classOf[CustomPage].getName) with IControler {

}